force-app/main/default/classes/DMLOperations.cls

```diff
@@ -128,11 +144,20 @@ public with sharing class DMLOperations {
      * @param list of Opportunities to be changed then upserted.
      */
     public static void upsertOpportunityList(List<Opportunity> opportunities) {
+
         // Loop through each Opportunity in the list
+        for (Opportunity opp : opportunities) {
             // Set the Opportunity's Stage to Qualification
+            opp.StageName = 'Qualification';
             // Set the Opportunity's Close Date to 3 months from today
+            opp.CloseDate = Date.today().addMonths(3);
             // Set the Opportunity's Amount to 50000
+            //*it is expecting an integer in the test class but the opportuntiy.amount field type is currency...?
```

Comment:
Currency fields resolve to Decimal in Apex. An Integer can be a Decimal, even if you dont use the decimal point.